test/01LessonOneTest.js

```diff
@@ -22,6 +22,8 @@ function greaterThan(big, small) {
 function lessThan(small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
+//iii
+
```

Comment:
@alexvazquez1220 Good job on the Pig Latin App! Could you finish out these lesson 1 tests?